docs/advanced-concepts/train-the-model-on-a-kubernetes-cluster-with-cml/.gitignore

```diff
@@ -1,3 +1,6 @@
+## Macos Specific
+.DS_Store
+
```

Comment:
As this file is already ignored in the main `.gitignore` file at the root level of this repository (https://github.com/csia-pme/a-guide-to-mlops/blob/16-minor-improvements/.gitignore#L23-L38) with all the others conflicting files for Linux and Windows, do you think it's necessary to add this to the guide as it's very specific to one OS?